dev-docs/bidders/escalax.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: bidder
+title: Escalax
+description: Prebid Escalax Bidder Adaptor
+biddercode: escalax
+usp_supported: true
+tcfeu_supported: true
```

Comment:
You cannot claim `tcf_supported` if you don't have `gvl_id` . TCF checks are performed against vendors and purposes.